Airports/EGPG/SMR/geo.txt

```diff
@@ -0,0 +1,153 @@
+N055.58.30.085 W003.58.24.135 N055.58.30.102 W003.58.24.141 smrWhite
+N055.58.30.102 W003.58.24.141 N055.58.30.394 W003.58.22.487 smrWhite
+; - CL
+N055.58.30.976 W003.58.19.188 N055.58.30.958 W003.58.19.182 smrWhite
+N055.58.30.958 W003.58.19.182 N055.58.30.665 W003.58.20.839 smrWhite
+N055.58.30.665 W003.58.20.839 N055.58.30.687 W003.58.20.852 smrWhite
+N055.58.30.687 W003.58.20.852 N055.58.30.976 W003.58.19.188 smrWhite
+; - CL
+N055.58.31.561 W003.58.15.888 N055.58.31.537 W003.58.15.875 smrWhite
+N055.58.31.537 W003.58.15.875 N055.58.31.248 W003.58.17.541 smrWhite
+N055.58.31.248 W003.58.17.541 N055.58.31.267 W003.58.17.551 smrWhite
+N055.58.31.267 W003.58.17.551 N055.58.31.561 W003.58.15.888 smrWhite
+; - CL
+N055.58.32.143 W003.58.12.593 N055.58.32.120 W003.58.12.584 smrWhite
+N055.58.32.120 W003.58.12.584 N055.58.31.825 W003.58.14.237 smrWhite
+N055.58.31.825 W003.58.14.237 N055.58.31.852 W003.58.14.252 smrWhite
+N055.58.31.852 W003.58.14.252 N055.58.32.143 W003.58.12.593 smrWhite
+; - 2
+N055.58.32.537 W003.58.09.673 N055.58.32.512 W003.58.09.663 smrWhite
+N055.58.32.512 W003.58.09.663 N055.58.32.485 W003.58.09.804 smrWhite
+N055.58.32.485 W003.58.09.804 N055.58.32.580 W003.58.09.862 smrWhite
+N055.58.32.580 W003.58.09.862 N055.58.32.611 W003.58.09.692 smrWhite
+N055.58.32.611 W003.58.09.692 N055.58.32.572 W003.58.09.471 smrWhite
+N055.58.32.572 W003.58.09.471 N055.58.32.583 W003.58.09.411 smrWhite
+N055.58.32.583 W003.58.09.411 N055.58.32.654 W003.58.09.454 smrWhite
+N055.58.32.654 W003.58.09.454 N055.58.32.672 W003.58.09.354 smrWhite
+N055.58.32.672 W003.58.09.354 N055.58.32.574 W003.58.09.295 smrWhite
+N055.58.32.574 W003.58.09.295 N055.58.32.544 W003.58.09.447 smrWhite
+N055.58.32.544 W003.58.09.447 N055.58.32.583 W003.58.09.688 smrWhite
+N055.58.32.583 W003.58.09.688 N055.58.32.583 W003.58.09.691 smrWhite
+N055.58.32.583 W003.58.09.691 N055.58.32.569 W003.58.09.761 smrWhite
+N055.58.32.569 W003.58.09.761 N055.58.32.528 W003.58.09.733 smrWhite
+N055.58.32.528 W003.58.09.733 N055.58.32.537 W003.58.09.673 smrWhite
+; - 5
+N055.58.32.737 W003.58.09.947 N055.58.32.642 W003.58.09.893 smrWhite
+N055.58.32.642 W003.58.09.893 N055.58.32.694 W003.58.09.553 smrWhite
+N055.58.32.694 W003.58.09.553 N055.58.32.764 W003.58.09.593 smrWhite
+N055.58.32.764 W003.58.09.593 N055.58.32.776 W003.58.09.522 smrWhite
+N055.58.32.776 W003.58.09.522 N055.58.32.711 W003.58.09.486 smrWhite
+N055.58.32.711 W003.58.09.486 N055.58.32.727 W003.58.09.377 smrWhite
+N055.58.32.727 W003.58.09.377 N055.58.32.823 W003.58.09.429 smrWhite
+N055.58.32.823 W003.58.09.429 N055.58.32.780 W003.58.09.698 smrWhite
+N055.58.32.780 W003.58.09.698 N055.58.32.710 W003.58.09.664 smrWhite
+N055.58.32.710 W003.58.09.664 N055.58.32.683 W003.58.09.818 smrWhite
+N055.58.32.683 W003.58.09.818 N055.58.32.751 W003.58.09.853 smrWhite
+N055.58.32.751 W003.58.09.853 N055.58.32.737 W003.58.09.947 smrWhite
+; - Taxline
+N055.58.32.200 W003.58.09.319 N055.58.31.944 W003.58.09.339 61166
```

Comment:
Define ```61166``` in [Colours.txt ](https://github.com/VATSIM-UK/UK-Sector-File/blob/main/Colours.txt)and change it to whatever you define it as:
```suggestion
N055.58.32.200 W003.58.09.319 N055.58.31.944 W003.58.09.339 smrPGTaxiway
```
Change ```smrPGTaxiway``` for whatever you define it as in [Colours.txt](https://github.com/VATSIM-UK/UK-Sector-File/blob/main/Colours.txt)